polkadot/runtime/parachains/src/configuration.rs

```diff
@@ -263,6 +263,8 @@ pub struct HostConfiguration<BlockNumber> {
 	pub minimum_backing_votes: u32,
 	/// Node features enablement.
 	pub node_features: NodeFeatures,
+	/// Bulk core count
+	pub bulk_core_count: u16,
```

Comment:
As mentioned on the ticket. I would suggest to re-use/rename the on-demand-core-count configuration, as it is replaced by this one.